pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -758,14 +758,26 @@ This binding is provided @racket[for-syntax].
 @defproc[(legacy-match-expander? [v any/c]) boolean?]]]{
 Predicates for values which implement the appropriate match expander
 properties.
+These bindings are provided @racket[for-syntax].
 }
 
 @defproc[(syntax-local-match-introduce [stx syntax?]) syntax?]{
 For backward compatibility only; equivalent to @racket[syntax-local-introduce].
 
+This binding is provided @racket[for-syntax].
+
 @history[#:changed "6.90.0.29" @elem{Made equivalent to @racket[syntax-local-introduce].}]}
 
 
+@defproc[(pattern-bound-identifiers #:legacy? [legacy? any/c #f] [pat syntax?]) (listof identifier?)]{
+Parse @racket[pat] as a pattern, and compute the identifiers bound by
+that pattern. If @racket[legacy?] is @racket[#f], use the legacy match
```

Comment:
Did you mean for this to be `#t` instead of `#f` here? Also, is there any reason users outside the Racket implementation would _need_ the legacy behavior? We could just leave this undocumented in the public API and only document it with a comment in the implementation.

Reply:
Yes, I mean `#t`.

Reply:
As for why you'd use it, the legacy syntax is public API. If you `(require mzlib/match)` you can still use it too. :)

Maybe there's no actual need to support it at all, though.

Reply:
Nobody's asked for this feature for legacy patterns, and if they do I think it'd probably be better to just make a separate copy of the function and export it from `mzlib/match`. No need to _preemptively_ make legacy APIs.